### DISCUSSION TITLE
Did not receive benefits after 72 hours

### DISCUSSION THREAD

**Discussion by HarryAllen1** [community/community#128614] [GitHub Education]
### Hi everyone,

I reapplied for student benefits a while ago and have waited over 72 hours, yet have still not received the benefits.
![image](https://github.com/community/community/assets/66224939/4b533301-29b6-4d9c-85b4-902cc2a2fac7)
![image](https://github.com/community/community/assets/66224939/ac3fbeae-467f-47b6-b3a7-8451fa0e6d7d)

Is there another step that I missed to get the GitHub Education benefits?

**Comment by davevad93** [comment]
Hi @HarryAllen1 ,

You haven't missed any steps, lately I have noticed that many users have had to wait a few days before they could access their benefits even though their request was already accepted. See this discussion #128457
